### PR TITLE
adds paths to charts source

### PR DESCRIPTION
**Pull request by YoussB**
Hey, 

- I added this due to having multiple failures in the main concourse pipeline due to frequent merges.
- in this PR I only restricted the git resource to the concourse chart, its dependency (Postgres), and the 2 metrics charts (not a direct dependency).

let me know what think?

### CODE REVIEW

**Comment by jchesterpivotal** [comment]
I'd suggest also limiting the depth.

File: pipeline.tpl.yml [hunk 1]
```diff
@@ -171,6 +171,11 @@ resources:
   source:
     branch: master
     uri: https://((github-token))@github.com/concourse/charts
+    paths:
+    - stable/concourse/
+    - stable/postgres/
+    - stable/grafana/
+    - stable/prometheus/
```
Comment (taylorsilva) [inline pipeline.tpl.yml]:
```suggestion
    depth: 5
    
```
Add `depth` so we avoid pulling in the entire history of the helm/charts repo.

Comment (jchesterpivotal) [reply pipeline.tpl.yml]: I think depth 5 should be OK. If not jump to a higher number (say 1024) and then use a binary search to pick a number (try 512, 256, 128 ...).

Comment (YoussB) [reply pipeline.tpl.yml]: sounds good!

**Pull request merged**